client/src/i18n/en.json

```diff
@@ -944,7 +948,33 @@
   },
   "VOUCHERS": {
     "SIMPLE": {
-      "TITLE" : "Transfer Slip"
+      "TITLE" : "Simple Vouchers"
+    },
+    "COMPLEX": {
+      "ADD_ITEMS"            : "Add item(s)",
+      "CASH_PAYMENT"         : "Cash Payment",
+      "CREATE_SUCCESS"       : "Successfully posted",
+      "CREATE_ERROR"         : "An error occured when post",
+      "CREDITOR"             : "Creditor (C)",
+      "DEBTOR"               : "Debitor (D)",
+      "DEB_CRED"             : "Debtor/Creditor",
+      "DESCRIPTION"          : "This Module allows you to make advanced journal transactions manually",
+      "ENTITY"               : "Entity (Debtor/Creditor)",
+      "ERROR_ACCOUNT"         : "Selected accounts are invalid",
+      "ERROR_AMOUNT"         : "Amount's values are invalid",
+      "ERROR_TOTALS"         : "The totals are not balanced",
+      "INVALID_VALUES"       : "There are invalid values detected",
+      "PATIENT_INVOICE"      : "Patient Invoice",
+      "PLACEHOLDER_CREDITOR" : "Enter a creditor",
+      "PLACEHOLDER_DEBTOR"   : "Enter a debtor",
+      "REFERENCE"            : "Reference",
+      "ROWS"                 : "Row(s)",
+      "SELECT_DOCUMENT_TYPE" : "Please select the document type",
+      "TITLE"                : "Advanced Journal Vouchers",
+      "TRANSACTION"          : "Transaction(s)",
+      "TYPE"                 : "Type",
```

Comment:
Is there a "FORM.LABELS.TYPE" key?  If not, this belongs as "FORM.LABELS.TYPE" instead of here.  Same with "TRANSACTION".